content/md/en/docs/reference/substrate-connect.md

```diff
@@ -0,0 +1,200 @@
+
+[What is a light client and why you should care? by Thibault Sardan](https://www.parity.io/what-is-a-light-client/)
+
+## Well known Networks
+
+* [Polkadot](https://polkadot.network/) (`polkadot`)
+* [Kusama](https://kusama.network/) (`ksmc33`)
+* [Westend](https://wiki.polkadot.network/docs/en/maintain-networks#westend-test-network) (`westend2`)
+* [Rococo](https://polkadot.network/rococo-v1-a-holiday-gift-to-the-polkadot-community/) (`rococo_v2_2`)
+
+[WellKnownChain enumeration docs](https://paritytech.github.io/substrate-connect/api/enums/connect_src.WellKnownChain.html)
+
+You have to provide a well-known chain name (`polkadot` for Polkadot, `ksmcc3` for Kusama, `westend2` for Westend and `rococo_v2_2` for Rococo). Note that these are the "real" names of the chains rather than the names they are more commonly known as (such as Kusama or Rococo). For example, "ksmcc3" is the name of Kusama. This is important for chains which have been hard forked. For example, "rococo_v2" and "rococo_v2_2" are two different chains.
+
+## Usage with PolkadotJS API (PolkadotJS provider)
+
+For using `substrate-connect` with the existing polkadotJS api, there is an rpc-provider that has `substrate-connect` already integrated. You can use that by installing in your app, the following packages:
+#### **@polkadot/rpc-provider**
+This package contains the `substrate-connect` rpc provider that integrates and uses substrate-connect:
+```bash
+yarn add @polkadot/rpc-provider
+```
+*or*
+```bash
+npm i @polkadot/rpc-provider
+```
+
+#### **@polkadot/api**
+
+This library provides a clean wrapper around all the methods exposed by a Polkadot/Substrate network client. It is used with the outcome of rpc-provider above
+
+```bash
+yarn add @polkadot/api
+```
+*or*
+```bash
+npm i @polkadot/api
+```
+
+Below are some examples that shows how the rpc-provider can be used simply with the already integrated networks (e.g. polkadot, kusama, westend, rococo), or with a custom chain by using its chainspec, or connecting to a parachain."
+
+*Simple usage (Using an integrated, suported chain)*
+```js
+import { ScProvider, WellKnownChain } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+// Create the provider for a known chain
+const provider = new ScProvider(WellKnownChain.westend2);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+*Simple usage (custom chain/chainspec)*
```

Comment:
```suggestion
### Connect to a custom chain specification using Polkadot-JS API
```

Reply:
That is not correct. User does not connect to a well known chains with polkadot JS API. User connects with Substrate connect provider and then communicates with that chain using the polkadot JS API.